hawkular-inventory-parent/hawkular-inventory-service/src/main/resources/eap-6-jmx-exporter.yml

```diff
@@ -0,0 +1,173 @@
+#
+# Copyright 2014-2017 Red Hat, Inc. and/or its affiliates
+# and other contributors as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+startDelaySeconds: 1
+lowercaseOutputName: true
+lowercaseOutputLabelNames: true
+whitelistObjectNames: [
+  "org.hawkular.agent:type=platform,subtype=*,name=*",
+  "jboss.as:path=*",
+  "jboss.as:deployment=*,subsystem=ejb3,stateless-session-bean=*",
+  "jboss.as:deployment=*,subsystem=ejb3,stateful-session-bean=*",
+  "jboss.as:deployment=*,subsystem=ejb3,message-driven-bean=*",
+  "jboss.as:deployment=*,subsystem=ejb3,singleton-bean=*",
+  "jboss.as:deployment=*,subsystem=undertow",
+  "jboss.as:deployment=*,subsystem=undertow,servlet=*",
+  "jboss.as:deployment=*,subdeployment=*,subsystem=ejb3,stateless-session-bean=*",
+  "jboss.as:deployment=*,subdeployment=*,subsystem=ejb3,stateful-session-bean=*",
+  "jboss.as:deployment=*,subdeployment=*,subsystem=ejb3,message-driven-bean=*",
+  "jboss.as:deployment=*,subdeployment=*,subsystem=ejb3,singleton-bean=*",
+  "jboss.as:deployment=*,subdeployment=*,subsystem=undertow",
+  "jboss.as:deployment=*,subdeployment=*,subsystem=undertow,servlet=*",
+  "jboss.as:subsystem=messaging-activemq,server=*,jms-queue=*",
+  "jboss.as:subsystem=messaging-activemq,server=*,jms-topic=*",
```

Comment:
I *think* the messaging subsystem is hornetq in EAP6 - need to use the correct names here